reactors/fcs-tasbe/example/fc.json

```diff
@@ -1,102 +1,105 @@
+				{
+					"name":                         "FITC-A",
+                                	"excitation_wavelength":        488, 
+                                	"emission_filter": {
+                                        	"type":"bandpass",
+                                        	"center": 525,
+                                        	"width": 20
+ 					}
+                                },
+				{
+					"name":  "PE-Tx-Red-YG-A",
+					"excitation_wavelength": 561,
+					"emission_filter": {
+						"type":"bandpass",
+						"center":610,
+						"width":20
+					}
+				}, 
+				{
+					"name":"Pacific Blue-A",
+					"excitation_wavelength":405,
+					"emission_filter": {
+						"type":"bandpass",
+						"center":450,
+						"width":50
+					}
 				}
-			},
-			{
-				"name": "PE-Tx-Red-YG-A",
-				"excitation_wavelength": 561,
-				"emission_filter": {
-					"type": "bandpass",
-					"center": 610,
-					"width": 20
-				}
-			},
-			{
-				"name": "Pacific Blue-A",
-				"excitation_wavelength": 405,
-				"emission_filter": {
-					"type": "bandpass",
-					"center": 450,
-					"width": 50
-				}
-			}
-		]
-	},
-	"color_model": {
-		"channels": [{
-				"name": "FITC-A",
-				"min": 2,
-				"label": "EYFP",
-				"calibration_file": "/data/controls/2012-03-12_EYFP_P3.fcs",
-				"chart_color": "y"
-			},
+			]
+		},
+	"color_model":{
+		"channels":[
+			{ 	"name": 			"FITC-A",
+				"min":				2,
+			  	"label":			"EYFP",
+			  	"calibration_file": 		"/data/controls/2012-03-12_EYFP_P3.fcs",
+			  	"chart_color":			"y"
+			 },
 
-			{
-				"name": "PE-Tx-Red-YG-A",
-				"min": 2,
-				"label": "mKate",
-				"calibration_file": "/data/controls/2012-03-12_mkate_P3.fcs",
-				"chart_color": "r"
-			},
-			{
-				"name": "Pacific Blue-A",
-				"min": 2,
-				"label": "EBFP2",
-				"calibration_file": "/data/controls/2012-03-12_ebfp2_P3.fcs",
-				"chart_color": "b"
-			}
-		],
-		"cross_file_pairs": [{
-				"channels": ["FITC-A", "PE-Tx-Red-YG-A", "Pacific Blue-A"],
-				"file": "/data/controls/2012-03-12_mkate_EBFP2_EYFP_P3.fcs"
+			{ 	"name": 			"PE-Tx-Red-YG-A",
+				"min":				2,
+				"label":			"mKate",
+				"calibration_file":		"/data/controls/2012-03-12_mkate_P3.fcs",
+				"chart_color":			"r"
 			},
-			{
-				"channels": ["FITC-A", "PE-Tx-Red-YG-A", "Pacific Blue-A"],
-				"file": "/data/controls/2012-03-12_mkate_EBFP2_EYFP_P3.fcs"
+			{	"name": 			"Pacific Blue-A",
+				"min":				2,
+				"label": 			"EBFP2",
+				"calibration_file" :		"/data/controls/2012-03-12_ebfp2_P3.fcs",
+				"chart_color":			"b"
 			}
 		],
-		"tasbe_config": {
-			"gating": {
-				"type": "auto"
+                "cross_file_pairs":[
+                       {
+                         "channels": ["FITC-A","PE-Tx-Red-YG-A","Pacific Blue-A"],
+                         "file": "/data/controls/2012-03-12_mkate_EBFP2_EYFP_P3.fcs"
+                       },
+                       {
+                         "channels": ["FITC-A","PE-Tx-Red-YG-A","Pacific Blue-A"],
```

Comment:
This is duplicate of line 57 - I believe you want to have a permutation here: Yellow, Blue, Red